### DISCUSSION TITLE
Using countUp without npm

### DISCUSSION THREAD

**Discussion by bsastregx** [inorganik/countUp.js#261] [Q&A] [answered]
Hi there!
is it possible to use countUp outside npm? 
In a simple html page.
If so, how?

Thanks a lot!

**Comment by simsal0r** [comment]
```
<script src="{% static 'your_path/countUp.umd.js' %}"></script>

<h1><span id="myTarget">100</span> likes</h1>

<script>
    var numAnim = new countUp.CountUp('myTarget', 2000);
    numAnim.start()
</script>

```

I think it would be quite useful for beginners, if something like this is higher up in the readme. I can offer to create a PR.

**Comment by inorganik** [comment]
Sorry to let this sit for so long. As you implied, it is already in the readme. But maybe we could include a line in the usage section, right under npm, like this:

## Usage:

**On npm** as `countup.js` Or, include the **[UMD script](https://github.com/inorganik/countUp.js#umd-module)** (see below)


... and then show an example script tag like you did, though without your framework syntax. Does that seem reasonable?

**Comment by inorganik** [comment, marked as answer]
I updated the README, this should be resolved now.
